### ISSUE TITLE
Diseñar footer

### ISSUE THREAD

**Issue by bossiernesto** [uqbar-project/uqbar-landing#11]
Hacer un footer que este mejor que el que tenemos ahora.

**Comment by bossiernesto** [comment]
Bump.. estaria bueno ver si queremos meter un footer o no

**Comment by PalumboN** [comment]
Sí, tal vez haya que poner alguna info legal o algo así (? Sino no me calentaría.

**Comment by bossiernesto** [comment]
Si +1 a lo que dice @PalumboN si no hay nada, creo que no vale la pena mucho agregar esta seccion.
